src/Ziggy.php

```diff
@@ -177,12 +177,23 @@ private function resolveBindings(array $routes): array
                     && $model === (new ReflectionMethod($model, 'getRouteKeyName'))->class;
 
                 // Avoid booting this model if it doesn't override the default route key name
-                $bindings[$parameter->getName()] = $override ? app($model)->getRouteKeyName() : 'id';
+                $bindings[$parameter->getName()] = $override
+                    ? $this->determineRouteKeyName($model)
+                    : 'id';
             }
 
             $routes[$name] = $scopedBindings ? array_merge($bindings, $route->bindingFields()) : $bindings;
         }
 
         return $routes;
     }
+
+    private function determineRouteKeyName(string $model): string
+    {
+        if (config()->has("ziggy.route_keys.$model")) {
+            return config("ziggy.route_keys.$model");
+        }
```

Comment:
What's an example of a `$model` you would put here?

Reply:
Outdated diff.